fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
```

Comment:
Nice. This would also mean some small changes in the tarFetcher class to define a boolean that would signal if we get the stream from the dockerhubFetcher or prepare it ourselves here in the unpack() method.